.githooks/pre-commit

```diff
@@ -3,6 +3,21 @@ cd "$(git rev-parse --show-toplevel)"
 ESLINT="client/node_modules/.bin/eslint"
 pwd
 
+staged=`git diff --name-only --cached --diff-filter=A --diff-filter=M | cat`
```

Comment:
Kevin already checks for staged files in line 26 in a more robust way. I would do something like

```
DIFF=($(git diff --cached --name-only --diff-filter=ACM))
STAGED_PYTHON = DIFF | grep '.py' # <-- Syntax not correct
```